internal/cmd/skupper/link/kube/link_status.go

```diff
@@ -0,0 +1,95 @@
+package kube
+
+import (
+	"context"
+	"fmt"
+	"github.com/skupperproject/skupper/client"
+	"github.com/skupperproject/skupper/internal/cmd/skupper/utils"
+	skupperv1alpha1 "github.com/skupperproject/skupper/pkg/generated/client/clientset/versioned/typed/skupper/v1alpha1"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
+	"text/tabwriter"
+)
+
+var (
+	linkStatusLong = `Display the status of links in the current site.`
+)
+
+type CmdLinkStatus struct {
+	Client    skupperv1alpha1.SkupperV1alpha1Interface
+	CobraCmd  cobra.Command
+	Namespace string
+}
+
+func NewCmdLinkStatus() *CmdLinkStatus {
+
+	skupperCmd := CmdLinkStatus{}
+
+	cmd := cobra.Command{
+		Use:     "status",
+		Short:   "Display the status",
+		Long:    linkStatusLong,
+		Example: "skupper link status",
+		PreRun:  skupperCmd.NewClient,
+		Run: func(cmd *cobra.Command, args []string) {
+			utils.HandleErrorList(skupperCmd.ValidateInput(args))
+			utils.HandleError(skupperCmd.Run())
+		},
+	}
+
+	skupperCmd.CobraCmd = cmd
+
+	return &skupperCmd
+}
+
+func (cmd *CmdLinkStatus) NewClient(cobraCommand *cobra.Command, args []string) {
+	cli, err := client.NewClient(cobraCommand.Flag("namespace").Value.String(), cobraCommand.Flag("context").Value.String(), cobraCommand.Flag("kubeconfig").Value.String())
+	utils.HandleError(err)
+
+	cmd.Client = cli.GetSkupperClient().SkupperV1alpha1()
+	cmd.Namespace = cli.Namespace
+}
+func (cmd *CmdLinkStatus) AddFlags() {}
```

Comment:
I haven't added it in the status command because it is not necessary to implement the WaitUntilReady function. The rest of the commands have the timeout flag added though: c1628c4cd999b79d2dbf7f8a137a5270bdb8a403